platformio.ini

```diff
@@ -80,7 +80,10 @@ upload_speed = 921600
 ; Uncomment below if you want to build for esp32
 ; Check your board name at https://docs.platformio.org/en/latest/platforms/espressif32.html#boards
 ; [env:esp32]
-; platform = espressif32 @ 6.1.0
+; platform = espressif32 @ 6.5.0
+; platform_packages =
```

Comment:
I don't rly like to have to use a alpha version. 
I dont think anyone did test the ADC seems they did add some breaking changes to it. 
https://github.com/espressif/arduino-esp32/issues/8796

Reply:
The breaking change in the ADC part is only about some pin routing functions. For the basic voltage reads, we're calling IDF functions directly and it should make no difference, at least in my tests it tells the correct voltage.

For using an alpha version it's because of the introduction of C++20 features. Also, as a byproduct, the later version of Arduino core would allow bringing in ESP32-C6 and later chips ( if they didn't bump major revision again )